bin/client/main.rs

```diff
@@ -75,7 +75,9 @@ impl TokenManager for PickYourAuth {
 async fn run() -> std::result::Result<(), ClientError> {
     let opts = opts::Opts::parse();
     // TODO: Allow log level setting outside of RUST_LOG (this is easier with this subscriber)
-    tracing_subscriber::fmt::init();
+    tracing_subscriber::fmt()
+        .with_ansi(atty::is(atty::Stream::Stderr))
```

Comment:
I just realized we didn’t change the client to use stderr for logging. Should just be able to copy paste from server

Reply:
You mean essentially this?

https://github.com/deislabs/bindle/blob/main/bin/server.rs#L157-L160

Reply:
Yep, exactly!

Reply:
@thomastaylor312 Can you check to make sure I understood correctly?

Reply:
Yep, that is correct!